ocm_python_wrapper/cluster.py

```diff
@@ -249,33 +249,79 @@ def addon_info(self):
             self.addon_name
         ).to_dict()
 
-    def validate_addon_parameters(self, parameters):
+    @staticmethod
+    def drop_unsupported_parameters(
+        parameters=[], cloud_provider="aws", product_id=None
```

Comment:
Cloud provider setting can use [this](https://github.com/RedHatQE/openshift-cluster-management-python-wrapper/pull/26/files#diff-3368f738eecbcf02482808ff710a68f13e75401e49898e0e7fa6d0a2b9aefdd3R219)

Reply:
see my above comment; this should be extracted in runtime when iterating over the conditions.